client-impl/simple-client/src/main/java/uk/gov/gchq/palisade/client/SimpleClient.java

```diff
@@ -37,18 +38,18 @@
 public class SimpleClient<T> {
     private final Serialiser<T> serialiser;
 
-    private final ServicesFactory services;
+    private final PalisadeService palisadeServices;
```

Comment:
I think this variable should be singular, e.g. palisadeService

Reply:
Done